### PR TITLE
Story3 whacking - moles disappear when clicked

### PR DESCRIPTION
**Pull request by snack-attack**

### CODE REVIEW

File: css/game.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+body {
+    background: #b8c77c;
+    font-family: 'Heebo';
+    font-size: 3rem;
+}
+
+.gamescreen {
+    border: solid 2px black;
+
```
Comment (sjr-mayden) [inline css/game.css]: Unnecessary whitespace

File: css/game.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+.table {
+    width: 83%;
+    position: relative;
+    z-index: 3;
+}
+
+.rat {
+    display: inline-block;
+    /*visibility: hidden;------->hides rat div */
```
Comment (sjr-mayden) [inline css/game.css]: Code that's commented out shouldn't be in a PR

File: css/splash.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+.heading {
+    font-size: 100px;
+    color: white;
+    transform: rotate(-20deg);
+    margin-top: 200px;
+    margin-left: 150px;
+    position: absolute;
+}
+.button {
```
Comment (sjr-mayden) [inline css/splash.css]: Missing new line

File: js/game.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+var rats = document.querySelectorAll('.rat');
+
+/**
+ * When you click on a rat, it disappears.
+ *
+ * @param {Node} rat The rat that is clicked.
+ */
+function whack(rat) {
+  rat.classList.add('hidden');
```
Comment (sjr-mayden) [inline js/game.js]: Semi colons when project/team style doesn't use them

File: js/game.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+rats.forEach(function(rat) {
+  rat.addEventListener('click', function() {
+    whack(rat);
```
Comment (sjr-mayden) [inline js/game.js]: Semi colons when project/team style doesn't use them

File: js/game.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  });
```
Comment (sjr-mayden) [inline js/game.js]: Semi colons when project/team style doesn't use them

File: js/game.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+});
```
Comment (sjr-mayden) [inline js/game.js]: Semi colons when project/team style doesn't use them

File: splash.html [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+<body class="body">
+    <div class="gamescreen">
+        <img class="image" src="images/exploding.svg">
+
+        <div class="buttons">
+            <h1 class="heading">whack-a-rat!</h1>
+            <button class="button">start</button>
+            <button class="button">instructions</button>
+        </div>
+
```
Comment (sjr-mayden) [inline splash.html]: Missing </div>

File: game.html [hunk 9]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+    <head>
```
Comment (sjr-mayden) [inline game.html]: Inconsistent indentation between game.html and splash.html

File: game.html [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+        <title>Whack-a-Rat</title>
+        <meta name="description" content="Mayden Academy JS Game">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/splash.css">
+        <link rel="stylesheet" href="css/game.css">
+
+    </head>
+    <body>
+        <!-- the whiteboard header with score + game timer -->
```
Comment (sjr-mayden) [inline game.html]: Inconsistent indentation between game.html and splash.html

**Review by sjr-mayden — DISMISSED** [review]
Testing functionality revealed issue with clicking of rats to make them disappear:

- Rats on bottom row cannot be made to disappear by clicking on their faces
- Rats on the top and bottom rows cannot be made to disappear by clicking on their tummies

Use "select an element in the page to inspect" in Chrome Dev Tools or command + a to see where desk divs are interfering with gameplay.

File: css/game.css [hunk 11]
```diff
@@ -0,0 +1,265 @@
...
+    margin-top: 100px;
+    border: solid 2px black;
+    position: relative;
+}
+
+.splash-content {
+    text-align: center;
+    position: absolute;
+    z-index: 10;
+    top:0;
```
Comment (mporam) [inline css/game.css]: missing space

File: game.html [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+                <h1 class="heading">whack-a-rat!</h1>
+                <button class="button start_button">start</button>
+                <button class="button instruction_button">instructions</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<!--INSTRUCTIONS-->
+<div class="instructions-container no_show">
```
Comment (mporam) [inline game.html:30]: please use gamescreen for the instructions screen so it doesnt move around

File: js/game.js [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+  rat.classList.add('hidden')
+}
+
+rats.forEach(function(rat) {
+  rat.addEventListener('click', function() {
+    whack(rat)
+  })
+})
+
+var rats = document.querySelectorAll('.rat')
```
Comment (mporam) [inline js/game.js]: this line is duplicate

Comment (mporam) [reply js/game.js]: infact, everything under this line is duplicated

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]
